src/python/pants/option/global_options.py

```diff
@@ -1380,7 +1380,9 @@ class BootstrapOptions:
         advanced=True,
         deprecation_start_version="2.15.0.dev2",
         removal_version="2.16.0.dev1",
-        removal_hint="Remote Auth plugins should use provide the function via the `remote_auth` entry point.",
+        removal_hint=softwrap(
+            "Remote auth plugins should now provide the function by implementing an entry point called remote_auth. If you are developing a plugin, switch to using an entry point. If you are only consuming a plugin from someone else, you can delete the remote_auth_plugin option and now only need the plugin to be included in [GLOBAL].plugins"
+        ),
```

Comment:
We want to avoid line length over 100. Search in this file for how softwrap is used with `"""`

Reply:
bad push. fixed.